### ISSUE TITLE
configuration on base class not used anymore

### ISSUE THREAD

**Issue by aklemp** [database-rider/database-rider#112]
Until version 1.2.9 (including) the configuration worked when having a test class with a base class containing the `@DBUnit` annotation. Version 1.2.10 introduced a new class `AnnotationUtils` for resolving such annotations. Unfortunately, it contains a bug for this specific case. Thus, the configuration is not used at all.

```
@DBUnit(caseSensitiveTableNames = true)
public abstract class MyBase {
}
public class MyTest extends MyBase {
    @Dataset("mydata.yml")
    @Test
    public void myTest() {
    }
}
```

I will provide a pull request with the fix shortly.

**Comment by rmpestano** [comment]
It should be fized now, can you try with v1.5.1-SNAPSHOT of db-rider? 

To enable database rider snapshots on your project, see here.

**Comment by aklemp** [comment]
Hi, the snapshot repository doesn't contain a rider core with a version >= 1.3.0. Are they located anywhere else?

**Comment by rmpestano** [comment]
Yes, I noticed that but now it is ok and you can try 1.5.1-SNAPSHOT.

Let me know if it fixes this issue so we can release the version.

**Comment by aklemp** [comment]
I've just tried the 1.5.1-SNAPSHOT and this bug is fixed.
